eng/common/scripts/Package-Properties.ps1

```diff
@@ -13,6 +13,7 @@ class PackageProps
     [string]$SdkType
     [boolean]$IsNewSdk
     [string]$ArtifactName
+    [string]$ReleaseDate
```

Comment:
```suggestion
    [string]$ReleaseStatus
```
Lets keep this named the same as we do in changelog so that is clearer that this isn't always a date.